web/modules/custom/dpl_filter_paragraphs/dpl_filter_paragraphs.module

```diff
@@ -2,9 +2,162 @@
 
 /**
  * @file
- * This file is left empty on purpose.
+ * DPL filtered Paragraphs.
```

Comment:
This huge chunk of code is mostly just the old code that is being taken back.

The differences:

This has been added:

```php
/**
 * Implements theme_preprocess_paragraph__NAME().
 *
 * @see _dpl_filter_paragraphs_prepare_filter_view()
 */
function dpl_filter_paragraphs_preprocess_paragraph__content_slider_automatic(array &$variables): void {
  $variables = _dpl_filter_paragraphs_prepare_filter_view($variables, 'content_paragraphs', 'slider', NULL);
}
```

and added to line 100:

```
'field_filter_content_types',
```